recipes/libxsmm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libxsmm" %}
+{% set version = "1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hfp/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 47c034e169820a9633770eece0e0fdd8d4a744e09b81da2af8c2608a4625811e
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+
+test:
+  commands:
+    - conda inspect linkages {{ name }}  # [not win]
```

Comment:
I think these are done automatically now.

Can you add a package specific test, even if it’s just checking the existence of the library. Without it’s quite easy to build an empty package by accident,

Reply:
I added extended tests.